src/hm-admin/hm-admin.component.ts

```diff
@@ -1,11 +1,16 @@
-import { Component } from '@angular/core';
+import {Component, Input} from '@angular/core';
+import {ConfigService} from './services/config/config';
 
 @Component({
   moduleId: module.id,
   selector: 'hm-admin-app',
   templateUrl: 'hm-admin.component.html',
   styleUrls: ['hm-admin.component.css']
 })
-export class HmAdminComponent {
+export class HmAdminComponent extends ConfigService {
```

Comment:
HmAdminComponent is a component, not a Service, so extend ConfigService is incorrect